cli/azd/cmd/middleware/hooks.go

```diff
@@ -74,7 +75,14 @@ func (m *HooksMiddleware) registerCommandHooks(
 	projectConfig *project.ProjectConfig,
 	next NextFn,
 ) (*actions.ActionResult, error) {
-	if len(projectConfig.Hooks) == 0 {
+
+	hooksDefinedAtInfraPath, err := ext.HooksFromFolderPath(
```

Comment:
Instead of overridding the `projectConfig.Hooks` in the middleware I would consider moving this over to the `ProjectConfig` itself so we know that as long as we have the `ProjectConfig` instance that everything is loaded / parsed correctly in a single place.